package.json

```diff
@@ -64,6 +64,7 @@
     "file-loader": "^1.1.11",
     "firebase": "^4.12.0",
     "flickity": "^2.0.11",
+    "npm": "^5.8.0",
```

Comment:
I'm guessing this got in here accidentally?

Reply:
i guess so 🤭